### PR TITLE
Update Direct Deposit integration tests

### PR DESCRIPTION
**Pull request by erikphansen**
## Description
Adds some integration tests using enzyme to make sure that Direct Deposit user interactions work correctly.

## Testing done
Local

## Screenshots
N/A

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/personalization/profile-2/tests/components/DirectDepositContent.unit.spec.jsx [hunk 1]
```diff
@@ -1,98 +1,242 @@
...
+  accountTypeSelect.simulate('change', {
+    target: { name: 'root_accountType', value: 'Savings' },
+  });
+
+  form.simulate('submit');
+}
+
+// Thanks Pivotal http://engineering.pivotal.io/post/react-integration-tests-with-enzyme/
+function asyncFlush() {
+  return new Promise(resolve => setTimeout(resolve, 0));
```
Comment (sanlouise) [inline src/applications/personalization/profile-2/tests/components/DirectDepositContent.unit.spec.jsx:84]: 👍

File: src/applications/personalization/profile-2/tests/components/DirectDepositContent.unit.spec.jsx [hunk 2]
```diff
@@ -1,98 +1,242 @@
...
+        current: 1,
+        highest: 3,
       },
-      vaProfile: {
-        paymentInformation: {
+    };
+
+    const wrapper = setUpWithInitialState(initialState);
+
+    expect(wrapper.html()).to.equal('');
```
Comment (sanlouise) [inline src/applications/personalization/profile-2/tests/components/DirectDepositContent.unit.spec.jsx:100]: This works, but I have used `isEmptyRender` in the past https://enzymejs.github.io/enzyme/docs/api/ReactWrapper/isEmptyRender.html

File: src/applications/personalization/profile-2/tests/components/DirectDepositContent.unit.spec.jsx [hunk 3]
```diff
@@ -1,98 +1,242 @@
...
+    wrapper.unmount();
+  });
+  it('should render nothing if the user does not have 2FA set up', () => {
+    initialState = createBasicInitialState();
+    initialState.user.profile.multifactor = false;
+
+    const wrapper = setUpWithInitialState(initialState);
+
+    expect(wrapper.html()).to.equal('');
```
Comment (sanlouise) [inline src/applications/personalization/profile-2/tests/components/DirectDepositContent.unit.spec.jsx:109]: idem to previous comment 😄

**Review by sanlouise — APPROVED** [review]
Great work adding these tests 👍

File: src/applications/personalization/profile-2/tests/components/DirectDepositContent.unit.spec.jsx (hunk 2 above)
Comment (erikphansen) [reply src/applications/personalization/profile-2/tests/components/DirectDepositContent.unit.spec.jsx:100]: Thanks, that is better. Updated!

**Pull request merged**